demisto_sdk/commands/common/hook_validations/graph_validator.py

```diff
@@ -206,14 +206,30 @@ def handle_invalid_toversion(
 
     @error_codes("GR103")
     def is_file_using_unknown_content(self):
+        """Validates that there is no usage of unknown content items.
+        The validation runs twice:
+        1. Cases where a warning should be raised - if the using content item is a test playbook/test script,
```

Comment:
maybe we can implement in the future the same thing also for the from_version and the to_version validations

Reply:
possible, but as long as it does not make noise it's less necessary.